commcare_connect/opportunity/tables.py

```diff
@@ -15,7 +15,7 @@ class OpportunityAccessTable(tables.Table):
 
     class Meta:
         model = OpportunityAccess
-        fields = ("user.name", "learn_progress", "visit_count")
+        fields = ("user.username", "learn_progress", "visit_count")
```

Comment:
this is fine for now, but long run we will want to fetch the name from connectid and use that rather than the username, since they are random strings and not user facing